magefiles/magefile.go

```diff
@@ -245,9 +301,15 @@ func UpdateTestfiles() {
 	TestUnit()
 }
 
+func TestOutline() {
```

Comment:
What does this target do?

Reply:
It creates a summary of behaviors we'd like to work on for BDD.  https://onsi.github.io/ginkgo/#creating-an-outline-of-specs